modules/utility.js

```diff
@@ -142,43 +144,31 @@ function checkPermission(msg, guildMember) {
 
 async function getChannelData(msg, args) {
     try {
-        let channelByID = args;
-        let channelID;
-        if (args === undefined) {
-            channelID = msg.channel.id;
-        } else {
-            channelID = channelByID;
-        }
-        let channel;
-        try {
-            channel = await msg.guild.channels.cache.get(channelID);
-        } catch (error) {
-            console.error();
-        }
-        return channel;
+        const channels = msg.guild.channels;
+        let channelId = msg.channel.id;
+
+        if (args) channelId = args;
+        return await channels.fetch(channelId);
```

Comment:
Kode berjalan baik ketika menggunakan `\channelinfo` dan `\channelinfo <id channel>`.
Namun saya lupa kalau channel juga bisa dimention, seperti contoh di bawah ini.
![Screenshot_1263](https://user-images.githubusercontent.com/87353624/149659438-922835d1-9452-469f-aebb-69c0767c154c.png)
(Gambar pertama tidak memunculkan error karena menggunakan kode yang saya rekomendasikan)

Dan jika menggunakan metode mention, maka akan muncul error seperti di bawah ini.
![Screenshot_1262](https://user-images.githubusercontent.com/87353624/149659451-6e759048-ab54-4a6a-a19e-b472d87e186c.png)

Saran:
```suggestion
    const mention = msg.mentions.channels.first();
    let channelId = msg.channel.id;

    if (args) {
      channelId = args;
      if (mention) channelId = mention.id;
    }
    return await msg.guild.channels.fetch(channelId);
```